google/cloud/completion_queue_test.cc

```diff
@@ -15,17 +15,26 @@
 #include "google/cloud/completion_queue.h"
 #include "google/cloud/future.h"
 #include "google/cloud/internal/default_completion_queue_impl.h"
+#include "google/cloud/status.h"
 #include "google/cloud/testing_util/async_sequencer.h"
 #include "google/cloud/testing_util/fake_completion_queue_impl.h"
 #include "google/cloud/testing_util/mock_async_response_reader.h"
 #include "google/cloud/testing_util/status_matchers.h"
 #include <google/protobuf/duration.pb.h>
 #include <google/protobuf/timestamp.pb.h>
 #include <gmock/gmock.h>
+#include <grpcpp/completion_queue.h>
+#include <grpcpp/support/async_unary_call.h>
+#include <grpcpp/support/status.h>
```

Comment:
Just FYI: I do not know whether these are public headers or not.  I have not found anything in gRPC documenting which headers are intended as part of the public API.